# goal
bool confirmed
recycle_msgs/LogItem log_item
---
# result
bool success
---
# feedback
